pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/login.md

```diff
@@ -4,7 +4,7 @@ $order: 1
 description: 'The first time you land on the page, you can see 2 comments and a login button. If you look for the login button in the code, you will find ...'
 ---
 
-The first time you land on the [page](https://ampbyexample.com/samples_templates/comment_section/preview/), you can see 2 comments and a login button.
+The first time you land on the [page]({{g.doc('/content/amp-dev/documentation/examples/documentation/Comment_Section.html', locale=doc.locale).url.path}}), you can see 2 comments and a login button.
```

Comment:
This should go to the preview.

Reply:
Done, now linked to: {{g.doc('/content/amp-dev/documentation/examples/interactivity-dynamic-content/comment_section/preview/index.html', locale=doc.locale).url.path}}